src/components/Loader/__stories__/Loader.stories.mdx

```diff
@@ -149,7 +149,7 @@ If there is a need for color customization, css `color` attribute of a parent co
 
 <Canvas>
   <Story name="Custom colors">
-    <div style={{ color: "var(--color-dark-red)" }}>
+    <div style={{ color: "#bb3354" }}>
```

Comment:
intentionally hard coded?

Reply:
Wasn't sure what to do here, cause we want to get rid of any usages of monday-ui-style tokens in the stories files, but I didn't want to add this color to the vibe-storybook-components just for this use case
WDYT?

Reply:
Let's create a tech debt ticket for it